documentation/jetty-documentation/src/main/asciidoc/operations-guide/annotations/chapter.adoc

```diff
@@ -71,21 +71,21 @@ Sometimes, you may have third party libraries on the container's classpath that
 In this case, use the `org.eclipse.jetty.server.webapp.ContainerIncludeJarPattern` context attribute to define which container jars and class directories to scan.
 The value of this attribute is a regular expression.
 
-Here's an example from a context xml file that includes any jar whose name starts with "foo-" or "bar-", or a directory named "classes":
+Here's an example from a context XML file that includes any jar whose name starts with `"foo-"` or `"bar-"`, or a directory named `"classes"`:
```

Comment:
This looks weird.

The tick + double-quote + text + double-quote + tick setup.
Why can't it just be tick + text + tick?

Looking at the existing documentation, we have the following usage counts

Style | Example | Count
---- | ---- | ----
double-quote + text + double-quote | `"text"` | 357
tick + text + tick | <code>&#x60;text&#x60;</code> | 2933
tick + double-quote + text + double-quote + tick | <code>&#x60;"text"&#x60;</code> | 22 instances
double-quote + tick + text + tick + double-quote | <code>"&#x60;text&#x60;"</code> | 9 instances